File: .sdk/tm/php/utility/PrepareBody.php
<?php
declare(strict_types=1);

// ArcgisHubWorldCountriesGeneralized SDK utility: prepare_body

class ArcgisHubWorldCountriesGeneralizedPrepareBody
{
    public static function call(ArcgisHubWorldCountriesGeneralizedContext $ctx): mixed
    {
        if ($ctx->op->input === 'data') {
            return ($ctx->utility->transform_request)($ctx);
        }
        return null;
    }
}

File: .sdk/tm/rb/utility/prepare_body.rb
# ArcgisHubWorldCountriesGeneralized SDK utility: prepare_body
module ArcgisHubWorldCountriesGeneralizedUtilities
  PrepareBody = ->(ctx) {
    ctx.op.input == "data" ? ctx.utility.transform_request.call(ctx) : nil
  }
end
